### DISCUSSION TITLE
Download the omnibug log with values

### DISCUSSION THREAD

**Discussion by ptkjayden** [MisterPhilip/omnibug#227] [Q&A] [answered]
You already have a download function that we can download(export to csv) the log.
but log has only Event type, Provider, Account, Request ID, Request URL, POST Data, Timestamp, Notes.
so could we download with variable and values like screen_shot?
(i attached the screen_shot) 

Thank you for supporting us.

![screen_shot](https://user-images.githubusercontent.com/110363772/232945121-f56aa28f-ff4c-4922-a4b0-1b52d4ff1137.png)

**Comment by MisterPhilip** [comment, marked as answer]
Sorry I missed this question! #226 is the same - basically there isn't a way to do this currently. I do plan on eventually adding this I just don't have a timeline for when it will be added.
